bookstore-jpa/pom.xml

```diff
@@ -15,7 +15,7 @@
         <dependency>
             <groupId>org.eclipse.persistence</groupId>
             <artifactId>eclipselink</artifactId>
-            <version>${eclipselink.version}</version>
+            <version>2.5.2</version>
```

Comment:
The version is defined in the parent-pom. You should adjust the number there, otherwise the modelgen-plugin has a different number